packages/core/src/apiProviders/utils.ts

```diff
@@ -104,8 +102,10 @@ const manageSucess = (
       type: 'success',
       position: 'bottom',
       bottomOffset: 20,
-      text1: `Success ${code}`,
-      text2: `${message ? message : 'Request successful'}.`,
+      text1: i18nProvider.i18n.t('Base_Success'),
+      text2: `${
+        message ? message : i18nProvider.i18n.t('Base_Request_Successful')
+      }.`,
```

Comment:
In case no message was given for request response, I think it could be better to have the same system as error toast by displaying the action instead of "Request successful" to give more informations on action.

"Successfully ${action}" / "${action} réussi(e)" -> This implies french translation should be something like "Mise à jour de la ligne" or "Ajout de la ligne"